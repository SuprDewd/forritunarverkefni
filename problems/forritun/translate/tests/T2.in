x=y
y=z
z=x
text:
x x y z y x z
