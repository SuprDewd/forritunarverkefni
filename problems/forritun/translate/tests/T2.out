y y z x z y x
